.github/workflows/test-suite.yml

```diff
@@ -88,37 +88,38 @@ jobs:
     - name: Show cache stats
       if: env.SELF_HOSTED_RUNNERS == 'true'
       run: sccache --show-stats
-  release-tests-windows:
-    name: release-tests-windows
-    needs: [check-labels]
-    if: needs.check-labels.outputs.skip_ci != 'true'
-    runs-on: ${{ github.repository == 'sigp/lighthouse' && fromJson('["self-hosted", "windows", "CI"]') || 'windows-2019'  }}
-    steps:
-    - uses: actions/checkout@v4
-    - name: Get latest version of stable Rust
-      if: env.SELF_HOSTED_RUNNERS == 'false'
-      uses: moonrepo/setup-rust@v1
-      with:
-          channel: stable
-          cache-target: release
-          bins: cargo-nextest
-      env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    - name: Install Foundry (anvil)
-      if: env.SELF_HOSTED_RUNNERS == 'false'
-      uses: foundry-rs/foundry-toolchain@v1
-      with:
-        version: nightly-ca67d15f4abd46394b324c50e21e66f306a1162d
-    - name: Install make
-      if: env.SELF_HOSTED_RUNNERS == 'false'
-      run: choco install -y make
-    - name: Set LIBCLANG_PATH
-      run: echo "LIBCLANG_PATH=$((gcm clang).source -replace "clang.exe")" >> $env:GITHUB_ENV
-    - name: Run tests in release
-      run: make nextest-release
-    - name: Show cache stats
-      if: env.SELF_HOSTED_RUNNERS == 'true'
-      run: sccache --show-stats
+# FIXME(das): disabled for now as the c-kzg-4844 `das` branch doesn't build on windows.
+#  release-tests-windows:
+#    name: release-tests-windows
+#    needs: [check-labels]
+#    if: needs.check-labels.outputs.skip_ci != 'true'
```

Comment:
We should be able to uncomment this now as we use the prod `c-kzg-4844` version on the `das` branch.

Reply:
Done